interfaces/acados_matlab_octave/dump_gnsf_functions.m

```diff
@@ -40,7 +40,7 @@ function dump_gnsf_functions(model)
 import casadi.*
 
 casadi_version = CasadiMeta.version();
-if ( strcmp(casadi_version(1:3),'3.5')) % require casadi 3.5 +
+if ( strcmp(casadi_version(1:3),'3.5') || strcmp(casadi_version(1:3),'3.6'))
 else % old casadi versions
     error('Please provide CasADi version or 3.5 to ensure compatibility with acados')
```

Comment:
The error message is not really a sentence. And this does not use the function `check_casadi_version()` because GNSF is more limiting regarding the casadi version? If so, could we add that to the error message?

Reply:
This file is just only here to be able to dump the GNSF structure detected system to then use it in Python.
CasADi introduced the serialize functionality in 3.5. afaik.
I'll improve the message though.